lib/main.dart

```diff
@@ -106,6 +107,8 @@ _setupApp() async {
   final metricClient = injector.get<MetricClientService>();
   metricClient.initService();
 
+  injector<CanvasClientService>().init();
```

Comment:
please be careful to call this `init` right after starting the app. Since I saw you called the `getDefaultAccount` which will create an account if not exist, then the onboarding will be automatically omitted.